pkg/doc/verifiable/credential.go

```diff
@@ -324,6 +324,20 @@ type CredentialTemplate func() *Credential
 // If not defined, JWT encoding is not tested.
 type PublicKeyFetcher func(issuerID, keyID string) (interface{}, error)
 
+// JWTDecoding defines if to decode VC from JWT
+type JWTDecoding int
```

Comment:
This exported  type doesn’t seem to be currently usable externally as it isn’t used as an argument to an option.

Reply:
True, no need to make it public.